shadowsocks-csharp/Model/Configuration.cs

```diff
@@ -37,9 +37,14 @@ public class Configuration
         public HotkeyConfig hotkey;
 
         private static readonly string CONFIG_FILE = "gui-config.json";
+
+        [JsonIgnore]
+        public bool updated = false;

```

Comment:
Should it be moved to UpdateChecker.cs ?
Reason:
1. This flag is set by comparing the local version and remote version.
2. It is only checked when checking updates.
3. No necessary for persistence and the flag is not widely used in SS.

Reply:
1. Binary version and config version, both are local.
2. Then manually download and replace won't trigger it.
3. Agree.

I'd like refactor UpdateChecker(maybe Configuration too?) to single instance pattern or static class before move it...

Reply:
I'll keep current behavior, refactor will in another PR.